src/vm/excep.cpp

```diff
@@ -7233,8 +7233,9 @@ EXTERN_C void JIT_WriteBarrier_Debug_End();
 EXTERN_C void FCallMemcpy_End();
 #endif
 
-static
-bool IsIPExcluded(UINT_PTR uControlPc)
+// Check if the passed in instruction pointer is in one of the
+// JIT helper functions.
+bool IsIPInJitHelper(UINT_PTR uControlPc)
```

Comment:
Ah, it seems we have the most difficult problem in the software development - picking the right name - again :-). 
The ShouldHandleJitHelperFault is a name of an already existing function, so we cannot use it.
How about `IsIPInMarkedJitHelper`? These are asm helpers that use `LEAF_END_MARKED` to indicate that the end is marked and so it can be used to figure out the function range.